src/ducks/transactions/actions/BillAction.jsx

```diff
@@ -135,9 +135,7 @@ const action = {
   name,
   icon,
   match: async (transaction, actionProps) => {
-    if (flag('reimbursements.tag')) {
-      return false
-    }
+    return false
```

Comment:
It means this action is not needed anymore, we can remove this file (AttachedDocsAction now shows the bills attached to a transaction)

Reply:
https://github.com/cozy/cozy-banks/issues/1754